docs/localdatetime.md

```diff
@@ -0,0 +1,1242 @@
+ldt = Temporal.LocalDateTime.from(savedUsingOldTzRules, { offset: 'reject' });
+// => RangeError: Offset is invalid for '2020-01-01T12:00' in 'America/Sao_Paulo'. Provided: -02:00, expected: -03:00.
+ldt = Temporal.LocalDateTime.from(savedUsingOldTzRules, { offset: 'use' });
+// => 2020-01-15T11:00-03:00[America/Sao_Paulo]
+// Evaluate DateTime value using old offset, which keeps UTC time constant as local time changes to 11:00
+ldt = Temporal.LocalDateTime.from(savedUsingOldTzRules, { offset: 'ignore' });
+// => 2020-01-15T12:00-03:00[America/Sao_Paulo]
+// Use current time zone rules to calculate offset, ignoring any saved offset
+ldt = Temporal.LocalDateTime.from(savedUsingOldTzRules, { offset: 'prefer' });
+// => 2020-01-15T12:00-03:00[America/Sao_Paulo]
+// Saved offset is invalid for current time zone rules, so use time zone to to calculate offset.
+```
+
+The `offset` option is ignored if an offset is not present in the input. In that case, the time zone is used to evaluate date/time values.
+
+The `disambiguation` option controls what time zone offset is used when the input time is ambiguous (as in the repeated clock hour after DST ends) or invalid due to offset changes skipping clock time (as in the skipped clock hour after DST starts):
+
+- `'compatible'` (the default): Acts like `'earlier'` for backward transitions and `'later'` for forward transitions.
+- `'earlier'`: The earlier of two possible times.
+- `'later'`: The later of two possible times.
+- `'reject'`: Throw a `RangeError` instead.
+
+When interoperating with existing code or services, `'compatible'` mode matches the behavior of legacy `Date` as well as libraries like moment.js, Luxon, and date-fns.
+This mode also matches the behavior of cross-platform standards like [RFC 5545 (iCalendar)](https://tools.ietf.org/html/rfc5545).
+
+During "skipped" clock time like the hour after DST starts in the Spring, this method interprets invalid times using the pre-transition time zone offset if `'compatible'` or `'later'` is used or the post-transition time zone offset if `'earlier'` is used.
+This behavior avoids exceptions when converting non-existent local time values to `Temporal.LocalDateTime`.
+
+For usage examples and a more complete explanation of how this disambiguation works and why it is necessary, see [Resolving ambiguity](./ambiguity.md).
+
+The `disambiguation` option is only used if there is no offset in the input, or if the offset is ignored by using the `offset` option as described above.
+If the offset in the input is used, then there is no ambiguity and the `disambiguation` option is ignored.
+
+> **NOTE**: The allowed values for the `thing.month` property start at 1, which is different from legacy `Date` where months are represented by zero-based indices (0 to 11).
+
+Example usage:
+
+```javascript
+ldt = Temporal.LocalDateTime.from('1995-12-07T03:24:30+02:00[Africa/Cairo]');
+ldt = Temporal.LocalDateTime.from('1995-12-07T03:24:30+02:00[Africa/Cairo][c=hijri]');
```

Comment:
`islamic` is the Unicode identifier, Hijri is one of the names it's called.

Reply:
fixed